packages/auth/src/types/requests.ts

```diff
@@ -43,6 +43,13 @@ export type SignInRequest<
 	options?: { serviceOptions?: ServiceOptions };
 };
 
+export type AuthProvider = 'Google' | 'Facebook' | 'Amazon' | 'Apple';
```

Comment:
Can we order this Alphabetically ?
```suggestion
export type AuthProvider =   'Amazon' | 'Apple'  | 'Facebook' | 'Google';
```